### PR TITLE
BNGP-3621: Non relevant persons: upload letter of consent and authority

### PR DESCRIPTION
**Pull request by jd-barnard**

### CODE REVIEW

File: packages/webapp/src/routes/__tests__/developer/consent-agreement-check.spec.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import constants from '../../../utils/constants.js'
 import { submitPostRequest } from '../helpers/server.js'
 
-const url = '/developer/check-consent-file'
+const url = '/developer/check-consent-file-test'
```
Comment (KiranVarma02) [inline packages/webapp/src/routes/__tests__/developer/consent-agreement-check.spec.js:4]: you added test to differentiate from main route? I think I prefer to use same constant as main route to not have to maintain path in 2 places.

Comment (KiranVarma02) [reply packages/webapp/src/routes/__tests__/developer/consent-agreement-check.spec.js:4]: I see you used main constant in other test files, so I guess this is a mistake!

Comment (jd-barnard) [reply packages/webapp/src/routes/__tests__/developer/consent-agreement-check.spec.js:4]: this test already existed with the route - but I've only added that route in now with this PR. There's a test in this file which expects a 404. On reflection that sounds like a bit of a side effect (the route didn't exist when that test was written). So I should probably just remove that test?

Comment (KiranVarma02) [reply packages/webapp/src/routes/__tests__/developer/consent-agreement-check.spec.js:4]: ahh ok, I think I would remove test

**Review by KiranVarma02 — APPROVED** [review]

**Review by KiranVarma02 — APPROVED** [review]

**Comment by shaundefra** [comment]
tested by Tajinder

**Pull request merged**